src/snowcli/cli/sql.py

```diff
@@ -62,5 +62,7 @@ def execute_sql(
         sql_text=sql,
         remove_comments=True,
     )
-    for result in results:
-        print_db_cursor(result)
+    if len(results) == 1:
```

Comment:
Is this check necessary? What would happen if we would  pass the one-item-long list to print_db_cursors?

Reply:
outdated :)